aio/content/examples/component-interaction/src/app/votetaker.component.ts

```diff
@@ -17,7 +17,7 @@ import { Component } from '@angular/core';
 export class VoteTakerComponent {
   agreed = 0;
   disagreed = 0;
-  voters = ['Narco', 'Celeritas', 'Bombasto'];
+  voters = ['Dr IQ', 'Celeritas', 'Bombasto'];
```

Comment:
For consistency with other usages:
```suggestion
  voters = ['Dr. IQ', 'Celeritas', 'Bombasto'];
```

Reply:
Thanks for the feedback. I did that in a pre-PR version of this fix, but I reverted it.
While reviewing the change, I ran into a bunch of other versions of the missing period in Dr. IQ's name (see below), some of which are also reflected in screen shots. So, I opted to leave this one as it was to prevent this from scope-creeping any further than it has already. This change should be a separate issue.
```
bobwatson-macbookpro:content bobwatson$ grep -R 'Dr IQ' * | grep -v stackblitz
examples/forms/src/app/hero-form/hero-form.component.ts:  model = new Hero(18, 'Dr IQ', this.powers[0], 'Chuck Overstreet');
examples/dynamic-component-loader/src/app/ad.service.ts:        { name: 'Dr IQ', bio: 'Smart as they come' }
examples/component-interaction/e2e/src/app.e2e-spec.ts:    const heroNames = ['Dr IQ', 'Magneta', 'Bombasto'];
examples/component-interaction/e2e/src/app.e2e-spec.ts:      const nonEmptyName = '"Dr IQ"';
examples/component-interaction/src/app/hero.ts:  {name: 'Dr IQ'},
examples/component-interaction/src/app/votetaker.component.ts:  voters = ['Dr IQ', 'Celeritas', 'Bombasto'];
examples/component-interaction/src/app/name-parent.component.ts:  // Displays 'Dr IQ', '<no name set>', 'Bombasto'
examples/component-interaction/src/app/name-parent.component.ts:  names = ['Dr IQ', '   ', '  Bombasto  '];
examples/dependency-injection/src/app/heroes/mock-heroes.ts:  { id: 18, isSecret: true,  name: 'Dr IQ' },
examples/router/src/app/heroes/mock-heroes.ts:  { id: 18, name: 'Dr IQ' },
```

Reply:
Thanks for additional info 👍